.config/dotnet-tools.json

```
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "microsoft.dotnet.darc": {
+      "version": "1.1.0-beta.25222.5",
```

Should this version equal the one in V.D.xml?

```suggestion
      "version": "1.1.0-beta.25225.3",
```